spec/acceptance/place_order_spec.rb

```diff
@@ -182,5 +183,68 @@
         )
       end
     end
+
+    context 'calculating the total price of items' do
+
+      it 'would return a number as its total' do
+        ordered_item = { id: '233', name: 'Bats', price: '12.00', quantity: '1' }
+        save_items_details.execute(item_details: ordered_item)
+
+        response = calculate_total_cost.execute
+        expect(response.is_a?(Numeric)).to be(true)
+      end
```

Comment:
maybe change this test to check that the response it equals 12

Reply:
There was a test below that does that